src/agora/dozf.cc

```diff
@@ -22,7 +22,8 @@ DoZF::DoZF(Config* config, int tid,
            Table<complex_float>& calib_ul_msum_buffer,
            PtrGrid<kFrameWnd, kMaxDataSCs, complex_float>& ul_zf_matrices,
            PtrGrid<kFrameWnd, kMaxDataSCs, complex_float>& dl_zf_matrices,
-           PhyStats* in_phy_stats, Stats* stats_manager)
+           PhyStats* in_phy_stats, Stats* stats_manager,
```

Comment:
It might be better for the class to make mat_logger_array optional.  Might be ok to skip this now, but keep it in mind for the future.